### DISCUSSION TITLE
Cause? 2 Corals TPUs -> KERNEL PANIC

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#3202] [Ask A Question]
I have 2 corals (usbTPUs) connected to my nuc. 
One of them is causing my NUC to HALT with the error code "KERNEL PANIC" ... 

Could this be a power issue or ? They are connected to USB3 ports only... 
Can I retrieve the logs afterwards to see what the issue is?

**Comment by blakeblackshear** [comment]
Does the kernel panic only when 2 are plugged in? Does it only happen when running frigate? I have definitely used 2 at the same time on my development machine without issue.

**Reply by Minglarn** [reply]
Only PANIC when running with 2. Not instant, takes a couple of days.. 
Never tested without running Frigate. But it's the same one every time. The white led flashes constantly when kernel stops.. 
How much does a TPU draw? Could it be to much for my little nuc?

**Comment by Minglarn** [comment]
Are the TPUs running at MAX? If so, is it possible to run it in standard performance?
